pre-train/reader/reader.py

```diff
@@ -17,63 +21,71 @@ def read_text(fn):
 class TextMelIDLoader(torch.utils.data.Dataset):
     
     def __init__(self, list_file, mean_std_file, shuffle=True):
-
+        '''
+        list_file: 3-column: (path, n_frames, n_phones)
+        mean_std_file: tensor loadable into numpy, of shape (feat_dims, 2)
```

Comment:
shape (2, feat_dims), i.e. [mean_row, std_row]

Reply:
addressed in dc8655c446cdfb4e81312f50d3b952258d2d06b1